lib/modules.nix

```diff
@@ -154,7 +159,9 @@ rec {
         let collected = collectModules
           (specialArgs.modulesPath or "")
           (modules ++ [ internalModule ])
-          ({ inherit lib options config specialArgs; } // specialArgs);
+          ({
+            inherit lib options config specialArgs;
+           } // specialArgs);
```

Comment:
Is this reformat intentional or a result from an intermediate change?

Reply:
It was the latter. I've restored it.